src/main/java/com/auctionapp/api/controller/AuctionController.java

```diff
@@ -50,4 +51,39 @@ public AuctionController(final AuctionService auctionService) {
 
 		return ResponseEntity.status(HttpStatus.OK).body(auctions);
 	}
+
+	@GetMapping("/default")
```

Comment:
All of the methods here can be one method with a parameter letting you know what kind of sorting is needed, instead of separate APIs.